### PR TITLE
New Article: Elegant solutions using Randomization

### PR DESCRIPTION
**Pull request by Dragonado**
I introduce some randomization techniques/concepts by solving and explaining a few problems.

### CODE REVIEW

File: _posts/2021-01-06-Elegant-Algorithms-Using-Randomization.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+description: 'Finding simple and fast solutions using randomized algorithms compared to deterministic algorithms'
+tags:
+- IEEE NITK
+- CompSoc
+- Algorithms
+- Randomization
+categories:
+- CompSoc
+github_username: 'Dragonado'
+---
```
Comment (adharshkamath) [inline _posts/2021-01-06-Elegant-Algorithms-Using-Randomization.md:16]: add  `use_math: true` here for Mathjax

**Review by adharshkamath — CHANGES_REQUESTED** [review]

File: _posts/2021-01-06-Elegant-Algorithms-Using-Randomization.md (hunk 1 above)
Comment (Dragonado) [reply _posts/2021-01-06-Elegant-Algorithms-Using-Randomization.md:17]: I just added it. I did not build the blog locally last time, I have done it now and it seems okay.

**Comment by adharshkamath** [comment]
Please resolve the merge conflict

**Comment by Dragonado** [comment]
Conflict in authors.yml has been resolved.

**Review by adharshkamath — APPROVED** [review]

**Comment by adharshkamath** [comment]
Remove the build files please

**Comment by Dragonado** [comment]
I have removed all build files. Only 2 files have changes that is authors.yml and my blog

**Review by adharshkamath — APPROVED** [review]

**Pull request merged**